monitor/services/src/main/java/org/datacleaner/monitor/configuration/TenantContextFactory.java

```diff
@@ -29,4 +31,6 @@
     public TenantContext getContext(TenantIdentifier tenant);
     
     public TenantContext getContext(String tenantId);
+
+    public Set<String> getAllTenantsName();
```

Comment:
On the content side we should make this an Iterator<String> because we can potentially be getting a VERY long list of tenants when we deploy this on datacleaner.org (15-20,000 registered users)